test/Table_Tests/src/In_Memory/Lossy_Conversions_Spec.enso

```diff
@@ -78,8 +78,8 @@ spec =
             Problems.assume_no_problems c6
             c6.to_vector.map .to_text . should_equal ([x+1, 1.0, x+2, "a", x+3].map .to_text)
 
-            # It should also work for from_vector_repeated:
-            c7 = Column.from_vector_repeated "X" [1.5, x+2, 100] 10
+            # It should also work for from_repeated_item:
+            c7 = Column.from_repeated_item "X" [1.5, x+2, 100] 10
             Problems.expect_only_warning Loss_Of_Integer_Precision c7
 
```

Comment:
Is this test even passing for you?

The second argument cannot be a Vector anymore. I think with the changes, this test doesn't make sense anymore - let's just remove it (ofc only the `from_repeated_item` case, others should stay obviously).

Reply:
Done